docs/en/cookbook/working-with-datetime.rst

```diff
@@ -94,7 +94,7 @@ the UTC time at the time of the booking and the timezone the event happened in.
 
         public function convertToDatabaseValue($value, AbstractPlatform $platform)
         {
-            if ($value instanceof \DateTime) {
+            if ($value instanceof \DateTimeInterface) {
                 $value->setTimezone(self::getUtc());
```

Comment:
Requires changing to `$value = $value->setTimezone(self::getUtc())`, as per `DateTimeImmutable`